src/app/results/results.component.html

```diff
@@ -1,3 +1,18 @@
+<head>
+
+<style type="text/css">
+.active {
+    color: black;
+    font-weight: bold;
+}
+</style>
+
+
```

Comment:
if you can write styles in corresponding  CSS files it will be really helpful for other developers to understand styles.